cartaya/1/task.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="This is my firs task from Leo">
+    <meta name="keywords" content="study,html5,css,task">
+    <meta name="author" content="Carlos Miguel Cartaya Leon">
+    <title>Task#1</title>
+    <link rel="stylesheet" type="text/css" href="style.css">
+</head>
+<body>
+    <section>
+        <div class="img">

```

Comment:
You need always use a convention to the names of all classes, it should create collisions.